jinahub/indexers/searcher/HnswlibSearcher/hnswlib_searcher.py

```diff
@@ -29,6 +29,7 @@ def __init__(
         dump_path: Optional[str] = None,
         traversal_paths: Iterable[str] = ('r',),
         is_distance: bool = True,
+        ignore_invalid_docs: bool = True,
```

Comment:
```suggestion
        check_embedding: bool = True,
```

Reply:
the parameter name`check_embedding=True` cannot indicate the action of filtering.  Could the users confuse this parameter?

Reply:
I'd propose `filter_invalid_docs`